modules/calib3d/src/calibinit.cpp

```diff
@@ -479,7 +479,7 @@ bool findChessboardCorners(InputArray image_, Size pattern_size,
 
     bool found = false;
 
-    const bool is_plain = (flags & CALIB_CB_PLAIN == CALIB_CB_PLAIN);
+    const bool is_plain = (flags & CALIB_CB_PLAIN) != 0;
```

Comment:
There is some unicode symbol before parenthesis:
```suggestion
    const bool is_plain = (flags & CALIB_CB_PLAIN) != 0;
```

Reply:
sorry about that
I just push the fix